CHANGELOG.md

```diff
@@ -1,9 +1,44 @@
 # Changelog
 
-## unreleased
-
-* [FEATURE] Remote-Write: Add sender and receiver support for [Remote Write 2.0-rc.2](https://prometheus.io/docs/specs/remote_write_spec_2_0/) specification #14395 #14427 #14444
-* [ENHANCEMENT] Remote-Write: 1.x messages against Remote Write 2.x Receivers will have now correct values for `prometheus_storage_<samples|histograms|exemplar>_failed_total` in case of partial errors #14444
+## 2.54.0-rc.0 / 2024-07-19
+
+Release 2.54 brings a release candidate of a major new version of [Remote Write: 2.0](https://prometheus.io/docs/specs/remote_write_spec_2_0/).
+This is experimental at this time and may still change.
+Remote-write v2 is enabled by default, but can be disabled via feature-flag `web.remote-write-receiver.accepted-protobuf-messages`.
+
+* [CHANGE] Remote-Write: `highest_timestamp_in_seconds` and `queue_highest_sent_timestamp_seconds` metrics now initialized to 0. #14437
+* [CHANGE] API: Split warnings from info annotations in API response #14327
+* [FEATURE] Remote-Write: Version 2.0 experimental, plus metadata in WAL via feature flag `metadata-wal-records` (defaults on). #14395,#14427,#14444
+* [FEATURE] PromQL: add limitk() and limit_ratio() aggregation operators #12503
+* [ENHANCEMENT] PromQL: Accept underscores in literal numbers, e.g. 1_000_000 for 1 million. #12821
+* [ENHANCEMENT] PromQL: Accept float literal numbers as durations (experimental). #9138
```

Comment:
And the other way around (duration are now also float literals).

Reply:
Does that mean I can say `5m * 3` and get 15m ?

Reply:
Maybe I should ask: when would a user benefit from "duration are now also float literals" ?

Reply:
> Does that mean I can say 5m * 3 and get 15m?
> Maybe I should ask: when would a user benefit from "duration are now also float literals" ?

You won't be able to do binary arithmetic in the context of a range vector selector, so `foo[5m * 3]` will not work. However, you'll be able to write things like:

* `predict_linear(foo[1h], 24h)` instead of `predict_linear(foo[1h], 24 * 3600)`
* `time() - my_timestamp > 10m` instead of `time() - my_timestamp > 10 * 60`

Reply:
Thanks, @juliusv, for explaining. (And I should note that static arithmetic in a range vector selector is a feature we would still like to implement, but one step after another…)